### ISSUE TITLE
UDT Project Dead?!

### ISSUE THREAD

**Issue by Streebor** [google/universal-dash-transmuxer#19]
I am not seeing any new commits from Google. The last meaningful commit was done in 2015. 

I am interested in using this lib in my iOS app, but I am hesitant to do it since git feels abandoned.

@SeawardT I was you respond so some issues. Can you shed some light on this?

Thank you!

**Comment by joeyparrish** [comment]
Yes, this project is dead.  I apologize for not remembering to archive it when we stopped supporting it.  It was many years ago.

This used to be part of the Widevine tech stack for iOS, along with some other closed-source components.  That was all replaced by Shaka Player Embedded: https://github.com/google/shaka-player-embedded

Please accept our apologies for the oversight.  If you are interested in the code, it's not going away, and you may fork it, make modifications, and distribute your fork as permitted by the license (Apache 2.0).
